plugins/actix-web/Cargo.toml

```diff
@@ -19,10 +19,15 @@ paperclip-macros = { path = "../../macros", version = "0.4.0", features = ["acti
 parking_lot = ">=0.10,<0.12"
 serde_json = "1.0"
 once_cell = "1.4"
+tinytemplate = { version = "1.1.0", optional = true }
+serde = { version = "1.0", optional = true }
+serde_qs = { version = "0.6", optional = true, features = ["actix"] }
```

Comment:
This is probably a bad merge. We don't need `serde_qs` here as we already have it in `core`?